app/components/SelectFilter.js

```diff
@@ -11,7 +11,7 @@ export default {
       </select>
       `,
   props: ['config', 'rows'],
-  emits: ['selected'],
+  emits: ['filterChange'],
```

Comment:
Choosing a consistent name for the emit for both filter components

Reply:
Good idea.